src/System.Management.Automation/engine/hostifaces/PSTask.cs

```diff
@@ -428,6 +438,22 @@ public void Start()
             _runspace.Name = string.Format(CultureInfo.InvariantCulture, "{0}:{1}", RunspaceName, s_taskId);
             _runspace.Open();
 
+            // Set current working directory on the runspace to be the same as the calling script.
```

Comment:
The comment is not accurate - we assign the location in constructor. We could move the comment where the constructor is called.